.github/workflows/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: Mark and close stale pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-stale: 14
+        days-before-close: 7
```

Comment:
Do we need another message for closing?

Reply:
The stale action doesn't provide for a closing message. The most recent comment in a PR being closed will be the stale message, so I think that's reasonable.

Is there something you'd like to add to the stale message?

Reply:
I think that is fine then. I was ideally hoping for one final message about "No action, so closing this", so no one wonders why their PR just got closed. However, as you said the last message will be the stale one, so that works.

Reply:
will a comment from the author count towards activity?

Reply:
The code uses `issue.updated_at`, so I believe any changes will count: https://github.com/actions/stale/blob/0d6a8e531449ecb97c9b6046b6b5d8eeac8b5115/src/main.ts#L101-L102